lib/puppet/provider/chocolateysource/windows.rb

```diff
@@ -176,7 +176,7 @@ def validate
 
   def flush
     args = []
-    opts = {}
+    opts = { sensitive: true }
```

Comment:
If we are setting opts here we should probably use it..

It looks like opts is actually redefined on line 202 for use later on 224...

but then on 231 we are ignoring opts and adding in the parameters again..

Reply:
Ah, have it this way because opts is redefined within an if statement.
Considered doing an else to have private passed but thought this was quicker.

Reply:
`opts` is a hash. It's odd that it is redefined and not added to.

e.g opts['new_property'] = 'new value'

Reply:
updated